docs/src/pages/customization/components/components.md

```diff
@@ -5,90 +5,28 @@
 As components can be used in different contexts, there are several approaches to this. Going from the narrowest use-case to the broadest, these are:
 
 1. [Specific variation for a one-time situation](#1-specific-variation-for-a-one-time-situation)
-1. [Dynamic variation for a one-time situation](#2-dynamic-variation-for-a-one-time-situation)
-1. [Specific variation of a component](#3-specific-variation-of-a-component) re-used in different contexts
+1. [Specific variations for reusable style overrides](#2-specific-variations-for-reusable-style-overrides)
+1. [Dynamic variation for a one-time situation](#3-dynamic-variation-for-a-one-time-situation)
 1. [Material Design variations](#4-material-design-variations) such as with the button component
```

Comment:
Does it even fit in this context?

```suggestion
```

Reply:
I was also a bit surprised to see it here, let's remove it, it has nothing to do with customization